serde_derive/src/de.rs

```diff
@@ -447,7 +447,30 @@ fn deserialize_seq(
         };
     }
 
+    let let_default = match *cattrs.default() {
+        attr::Default::Default => {
+            Some(
+                quote!(
+                let __default: Self::Value = _serde::export::Default::default();
+            ),
```

Comment:
Can you indent this line and the one above + the same thing in the `Path` match arm